Components/Comments.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react"
+import { getArticleComments } from "../src/utils/api"
+
+
+const Comments = ({articleId}) => {
+
+    const [comments, setComments] = useState([])
+
```

Comment:
Can you please add a loading state for the comments and add a loading message ⌛

Reply:
yup, thank you